int_array_param:
- 1
- 2
- 3
string_array_param:
- 'A0: 1.0, 0.0, 0.0'
- 'A1: 0.0, 0.0, 0.0'
- 'A2: 0.0, 0.0, 0.0'
